plugins/modules/public_cloud_private_network_info.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, division, print_function
+
+from ansible.module_utils.basic import AnsibleModule
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: public_cloud_private_network_info
+
+short_description: Get information about private networks.
+
+description:
+    - Get the openstackId for a private network depending on the region.
+
+requirements:
+    - ovh >= 0.5.0
+
+options:
+    service_name:
+        required: true
+        description: The service name
+    private_network:
+        required: true
+        description: The OVH private network
+    region:
+        required: true
+        description: The region where to lookup for network
+"""
+
+EXAMPLES = r"""
+  - name: Get openstackId
```

Comment:
`Get openstackId of the network` or something like that

Reply:
I would even say `Get openstackId of the network in this region` if its value is region specific

Reply:
Fixed, thaks!